app/controllers/api/enterprise_fees_controller.rb

```diff
@@ -6,9 +6,9 @@ def destroy
       authorize! :destroy, enterprise_fee
 
       if enterprise_fee.destroy
-        render text: I18n.t(:successfully_removed), status: :no_content
```

Comment:
render text with .to_json is equal to render json
but is it equivalent without the to_json like here? Does the client code handles this correctly?

Reply:
Hmm... yeah, we need to verify this and possibly change the format a bit?

Reply:
Like: `{ message: I18n.t(:successfully_removed) }`

Reply:
> render text with .to_json is equal to render json but is it equivalent without the to_json like here?

I think `render json: @object` does something equivalent to  `render text: @object.to_json`, yes.

> Does the client code handles this correctly?

This issue is somewhat complicated... https://stackoverflow.com/a/22361229/9235874

But it looks like browsers regard all these values as valid JSON. So if we're already rendering a string as a response in a JSON endpoint, I think we'll get the same result.

Reply:
There are two cases:
- with to_json: "I think render json: @object does something equivalent to render text: @object.to_json, yes."

I think we need to test and be sure it works the same way.


- without to_json:

I think we need to test every case auto or manually and see if we are not breaking anything.